drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -1322,10 +1309,64 @@ local hue = Driver("hue",
       else
         return false
       end
+    end,
+    update_bridge_netinfo = function(self, bridge_id, bridge_info)
+      if self.joined_bridges[bridge_id] then
+        local bridge_device = self:get_device_by_dni(bridge_id)
+        if not bridge_device then
+          log.warn_with({ hub_logs = true },
+            string.format(
+              "Couldn't locate bridge device for joined bridge with DNI %s",
+              bridge_id
+            )
+          )
+          return
+        end
+
+        if bridge_info.ip ~= bridge_device:get_field(Fields.IPV4) then
+          update_bridge_fields_from_info(self, bridge_info, bridge_device)
+          local maybe_api_client = bridge_device:get_field(Fields.BRIDGE_API)
+          local maybe_api_key = bridge_device:get_field(Fields.API_KEY) or Discovery.api_keys[bridge_id]
+          local maybe_event_source = bridge_device:get_field(Fields.EVENT_SOURCE)
+          local bridge_url = "https://" .. bridge_info.ip
+
+          if maybe_api_key then
+            if maybe_api_client then
+              maybe_api_client:update_connection(bridge_url, maybe_api_key)
+            end
+
+            if maybe_event_source then
+              maybe_event_source:close()
+              bridge_device:set_field(Fields.EVENT_SOURCE, nil)
+              do_bridge_network_init(self, bridge_device, bridge_url, maybe_api_key)
+            end
+          end
+        end
+      end
+    end,
+    get_device_by_dni = function(self, dni)
+      local device_id = self.datastore.dni_to_device_id[dni]
```

Comment:
Have you considered using the term `uuid` instead of `device_id`? I find `device_id` and `device_network_id` difficult to keep separate in my head.

Reply:
I thought about it but it "seems" to me that internally we have a convention that has overloaded the acronym "UUID" to mean "User UUID". So I wanted to keep with our conversational and organizational conventions.

I'm not against changing that though.

Reply:
the arg to `get_device_info` is named `uuid`, at least